README.md

```diff
@@ -24,4 +24,9 @@ This project can be open in android studio and built from there.
 
 1. Download apk from https://github.com/gehee/FPVue_android/releases.
 2. Run `adb install FPVue_android_0.14.1.apk` from the command line.
-3. Download [gs.key](https://github.com/gehee/FPVue_android/raw/main/app/src/main/assets/gs.key), rename it to drone.key and copy it on the air side.
\ No newline at end of file
+3. Download [gs.key](https://github.com/gehee/FPVue_android/raw/main/app/src/main/assets/gs.key), rename it to drone.key and copy it on the air side.
+
+## OSD feature.
+- Press OSD Config to show/hide items. At the first time, items will appear at left top corner and overlap each other, let's drag and drop them at where you want.
```

Comment:
- Press OSD Config to show/hide items. At first, items will all appear at the top left corner. They can be drag and dropped to the desired position.